snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -338,19 +338,19 @@ def mark_pull_done(self):
         part_build_packages = self._part_properties.get('build-packages', [])
         build_packages = repo.Repo.get_installed_build_packages(
             part_build_packages)
-        self.build_packages = []
+        ann_build_packages = []
```

Comment:
What does `ann` mean? Annotated? Perhaps `versioned_build_packages` would be better?